internal/webhookcert/webhook_cert_test.go

```diff
@@ -268,28 +318,43 @@ func TestReuseExistingCertificate(t *testing.T) {
 	}(certDir)
 
 	config := Config{
-		CertDir:      certDir,
-		ServiceName:  webhookService,
-		CASecretName: caBundleSecret,
-		WebhookName:  webhookName,
+		CertDir:               certDir,
+		ServiceName:           webhookService,
+		CASecretName:          caBundleSecret,
+		ValidatingWebhookName: validatingWebhookNamespacedName,
+		MutatingWebhookName:   mutatingWebhookNamespacedName,
 	}
 
 	err = EnsureCertificate(context.TODO(), client, config)
 	require.NoError(t, err)
 
 	var newValidatingWebhookConfiguration admissionregistrationv1.ValidatingWebhookConfiguration
-	err = client.Get(context.Background(), types.NamespacedName{Name: name}, &newValidatingWebhookConfiguration)
+	err = client.Get(context.Background(), config.ValidatingWebhookName, &newValidatingWebhookConfiguration)
 	require.NoError(t, err)
 
 	err = EnsureCertificate(context.TODO(), client, config)
 	require.NoError(t, err)
 
 	var updatedValidatingWebhookConfiguration admissionregistrationv1.ValidatingWebhookConfiguration
-	err = client.Get(context.Background(), types.NamespacedName{Name: name}, &updatedValidatingWebhookConfiguration)
+	err = client.Get(context.Background(), config.ValidatingWebhookName, &updatedValidatingWebhookConfiguration)
 	require.NoError(t, err)
 
 	require.Equal(t, newValidatingWebhookConfiguration.Webhooks[0].ClientConfig.CABundle,
 		updatedValidatingWebhookConfiguration.Webhooks[0].ClientConfig.CABundle)
 	require.Equal(t, newValidatingWebhookConfiguration.Webhooks[1].ClientConfig.CABundle,
 		updatedValidatingWebhookConfiguration.Webhooks[1].ClientConfig.CABundle)
+
+	var newMutatingWebhookConfiguration admissionregistrationv1.MutatingWebhookConfiguration
+	err = client.Get(context.Background(), config.MutatingWebhookName, &newMutatingWebhookConfiguration)
+	require.NoError(t, err)
+
+	err = EnsureCertificate(context.TODO(), client, config)
+	require.NoError(t, err)
```

Comment:
We don't need to execute the `EnsureCertificate()` func for the third time here. We can remove this part. And the 3 
lines for fetching the `newMutatingWebhookConfiguration` can be executed directly after fetching the `newValidatingWebhookConfiguration`

Reply:
redundant call removed